client/components/relatedProducts/RelatedProducts.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import axios from 'axios';
+
+import Card from './Card';
+
+import '../../css/relatedProducts/RelatedProducts.scss';
+
+const RelatedProducts = ({ productId }) => {
+  const [products, setProducts] = useState([]);
+
+  useEffect(() => {
+    // make a GET request to /api/products/:product_id/related
+    axios.get(`/api/products/${productId}/related`)
+      .then(({ data }) => data)
+      .then((ids) => ids.map((id) => axios.get(`/api/products/${id}`)))
+      .then((promises) => {
```

Comment:
Like earlier comment you definitely want error handling here. Also depending no robustness you can use promise.allSettled but that is an optional later improvement. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled

Reply:
Cool 😎  Was able to add this change as well.